src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1445,21 +1445,21 @@ f1_csscslc_pcsircs,43,1994,Line 43,,,,
 f1_csscslc_pcsircs,44,1994,Line 44,,,,
 f1_csscslc_pcsircs,45,1994,Line 45,,,,
 f1_csscslc_pcsircs,46,1994,TOTAL,,,,
-f1_dacs_epda,1,1994,Intangible Plant,,,,
-f1_dacs_epda,2,1994,Steam Production Plant,,,,
-f1_dacs_epda,3,1994,Nuclear Production Plant,,,,
-f1_dacs_epda,4,1994,Hydraulic Production Plant-Conventional,,,,
-f1_dacs_epda,5,1994,Hydraulic Production Plant-Pumped Storage,,,,
-f1_dacs_epda,6,1994,Other Production Plant,,,,
-f1_dacs_epda,7,1994,Transmission Plant,,,,
-f1_dacs_epda,8,1994,Distribution Plant,,,,
-f1_dacs_epda,9,1994,General Plant,,,,
-f1_dacs_epda,10,1994,Common Plant-Electric,,,,
-f1_dacs_epda,11,1994,TOTAL,,,,
-f1_dacs_epda,9,2006,Regional Transmission and Market Operation Plant,,,,
-f1_dacs_epda,10,2006,General Plant,,,,
-f1_dacs_epda,11,2006,Common Plant-Electric,,,,
-f1_dacs_epda,12,2006,TOTAL,,,,
+f1_dacs_epda,1,1994,Intangible Plant,reported_value,intangible_plant,,
+f1_dacs_epda,2,1994,Steam Production Plant,reported_value,steam_production_plant,,
+f1_dacs_epda,3,1994,Nuclear Production Plant,reported_value,nuclear_production_plant,,
+f1_dacs_epda,4,1994,Hydraulic Production Plant-Conventional,reported_value,hydraulic_production_plant_conventional,,
+f1_dacs_epda,5,1994,Hydraulic Production Plant-Pumped Storage,reported_value,hydraulic_production_plant_pumped_storage,,
+f1_dacs_epda,6,1994,Other Production Plant,reported_value,other_production_plant,,
+f1_dacs_epda,7,1994,Transmission Plant,reported_value,transmission_plant,,
+f1_dacs_epda,8,1994,Distribution Plant,reported_value,distribution_plant,,
+f1_dacs_epda,9,1994,General Plant,reported_value,general_plant,,
+f1_dacs_epda,10,1994,Common Plant-Electric,reported_value,common_plant_electric,,
+f1_dacs_epda,11,1994,TOTAL,calculated_value,total,,
+f1_dacs_epda,9,2006,Regional Transmission and Market Operation Plant,reported_value,regional_transmission_market_plant,,
+f1_dacs_epda,10,2006,General Plant,reported_value,general_plant,,
+f1_dacs_epda,11,2006,Common Plant-Electric,reported_value,common_plant_electric,,
+f1_dacs_epda,12,2006,TOTAL,calculated_value,total,,
```

Comment:
Yeah this is what I was initially a little confused about, because I couldn't find any other tables that were doing quite this same thing. The row numbers correspond directly to the categories in the `functional_classification` axis, other than the `total` row

Reply:
Okay, then I think you need to set the `dbf_only` column equal to TRUE in here to indicate that there's a DBF value that doesn't show up in the XBRL data.